### PR TITLE
cpu/stm32l1: cleaned up UART pin configuration slightly

### PR DESCRIPTION
**Pull request by haukepetersen**

### CODE REVIEW

File: cpu/stm32l1/include/periph_cpu.h [hunk 1]
```diff
@@ -82,6 +82,17 @@ typedef enum {
 } gpio_af_t;
 
 /**
+ * @brief   Configure the alternate function for the given pin
+ *
+ * @note    This is meant for internal use in STM32F4 peripheral drivers only
```
Comment (thomaseichinger) [inline cpu/stm32l1/include/periph_cpu.h]: s/F4/L1/

**Comment by thomaseichinger** [comment]
One minor documentation comment. ACK when Travis passed.

**Comment by haukepetersen** [comment]
fixed doxygen, waiting for Travis

**Comment by thomaseichinger** [comment]
We missed it, please rebase.

**Comment by thomaseichinger** [comment]
Closed in favour of #4070.

**Pull request closed without merging**